lib/kafka/datadog.rb

```diff
@@ -96,8 +96,8 @@ class StatsdSubscriber < ActiveSupport::Subscriber
       private
 
       %w[increment histogram count timing gauge].each do |type|
-        define_method(type) do |*args|
-          emit(type, *args)
+        define_method(type) do |*args, **kwargs|
```

Comment:
This change was needed to support ruby 3 due to https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/.